core/src/model.jl

```diff
@@ -360,5 +361,7 @@ function solve!(model::Model)::Model
         SciMLBase.solve!(integrator)
     end
     check_error!(integrator)
+    comptime = canonicalize(Millisecond(round(Int, comptime_s * 1000)))
+    @info "Computation time: $comptime"
```

Comment:
It this captured by our logging system (and thus ends up in the log file)?

Reply:
Yes, I checked. And there are tests for this for other log messages.